src/app/ui/components/skeleton-loader/skeleton-loader.tsx

```diff
@@ -0,0 +1,56 @@
+import { css } from 'leather-styles/css';
+import { Box } from 'leather-styles/jsx';
+
+interface SkeletonLoaderProps {
+  children?: React.ReactNode;
+  isLoading: boolean;
+  variant:
+    | 'accountCardTotalBalance'
+    | 'accountItemTotalBalance'
+    | 'cryptoAssetItemTokenBalance'
+    | 'cryptoAssetItemUsdBalance';
+}
+
+const variantStyles = {
+  accountItemTotalBalance: {
+    height: '20px',
+  },
+  cryptoAssetItemTokenBalance: {
+    width: '126px',
+  },
+  cryptoAssetItemUsdBalance: {
+    width: '78px',
+  },
+  accountCardTotalBalance: {
+    width: '200px',
+    height: '38px',
+  },
```

Comment:
This breaks separation of concerns. A generic component like this shouldn't know about all these specific use-cases within the wallet. Will list just grow every new use we have? IMO this is a really important rule not to break, and easy to avoid.

Why not make it configurable by passing width/height. Or, perhaps better, given that the skeleton loader is there to replace areas of text, why not pass the `textStyle` that it's masking and infer the height from that? And also pass a length for how many chars long it should be? 

```ts
<SkeletonLoader textStyle="header.01" textLength={20} />
```

Reply:
Yep you're right, let's skip the textstyle approach, as we don't want to assume it's only used for text.

To avoid the duplication, we should make wrapper components

`AccountCardSkeletonLoader` contains `SkeletonLoader` with some predefined widths and heights. That's a more flexible approach than baking in these concerns to the loader itself.

Reply:
don't think we need any additional wrapper either. just pass width/height and that's it